tailwind.config.ts

```diff
@@ -1,12 +1,16 @@
 import type { Config } from "tailwindcss";
 
 const config: Config = {
+  corePlugins: {
```

Comment:
What is this for?

Reply:
twas from combining the two different exports in tailwind.config.ts in font PR